pkg/cri/constants/constants.go

```diff
@@ -16,11 +16,53 @@
 
 package constants
 
-// TODO(random-liu): Merge annotations package into this package.
```

Comment:
Just removing this comment is probably enough. Don't see any reason to merge the packages anymore.